contracts/contract/node/RocketNodeDeposit.sol

```diff
@@ -52,9 +60,11 @@ contract RocketNodeDeposit is RocketBase, RocketNodeDepositInterface {
         require(depositType != MinipoolDeposit.None, "Invalid node deposit amount");
         // Node is trusted
         if (rocketDaoNodeTrusted.getMemberIsValid(msg.sender)) {
-            // If creating an unbonded minipool, check current unbonded minipool count
+            // If creating an unbonded minipool, check current unbonded minipool count and current fee > 80% of max
             if (depositType == MinipoolDeposit.Empty) {
                 require(rocketDaoNodeTrusted.getMemberUnbondedValidatorCount(msg.sender) < rocketDaoNodeTrustedSettingsMembers.getMinipoolUnbondedMax(), "Trusted node member would exceed the amount of unbonded minipools allowed");
+                uint256 maxFee = rocketDAOProtocolSettingsNetwork.getMaximumNodeFee();
+                require(nodeFee > maxFee.mul(rocketDaoNodeTrustedSettingsMembers.getMinipoolUnbondedMinFee()).div(1 ether), "Trusted node member can only create an unbonded minipool if current fee > 80% of maximum fee");
```

Comment:
`rocketDaoNodeTrustedSettingsMembers.getMinipoolUnbondedMinFee()` if set to 0 (meaning they could make them any time), this would permanently throw when dividing by 0?

Also best to remove 80% from the revert message as that value may change.

Reply:
There is no possible div by 0. It's `maxFee * unbondedMinFee / 1 ether`. If unbondedMinFee is set to 0 then this equation will evaluate to 0.